viewing_party/party.py

```diff
@@ -1,23 +1,109 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = {}
+    if title and genre and rating: 
+        movie["title"] = title

```

Comment:
I suggest moving the `movie` declaration inside the if-statement so that we don't allocate space for `movie` unless we know it will be used.
```suggestion
    if title and genre and rating: 
        movie = {}
        movie["title"] = title
```